frontend/controller/actions/group.js

```diff
@@ -180,8 +182,11 @@ export default (sbp('sbp/selectors/register', {
             ringLevel: Number.MAX_SAFE_INTEGER,
             permissions: [GIMessage.OP_KEY_REQUEST],
             meta: {
-              quantity: 60,
-              expires: Date.now() + DAYS_MILLIS * INVITE_EXPIRES_IN_DAYS.ON_BOARDING,
+              quantity: MAX_GROUP_MEMBER_COUNT,
+              ...(INVITE_EXPIRES_IN_DAYS.ON_BOARDING && {
+                expires:
+                await sbp('chelonia/time') + DAYS_MILLIS * INVITE_EXPIRES_IN_DAYS.ON_BOARDING
```

Comment:
Before we had `Date.now()` (unix timestamp in millis) here, and now we have what I think is unix timestamp in seconds - so a mismatch